uber/models.py

```diff
@@ -1215,7 +1215,11 @@ def total_cost(self):
 
     @property
     def amount_unpaid(self):
-        return max(0, self.total_cost - self.amount_paid)
+        if self.paid == c.PAID_BY_GROUP:
```

Comment:
This is the other part of the change that makes things work - we want to make sure attendees aren't charged for their badges if they're part of a group. The max(0, X) is used because some group's badges may be free.